jenkins-integration/beaker/install/shared/23_install_system_gems.rb

```diff
@@ -0,0 +1,17 @@
+require 'json'
+
+test_name 'Install System Gems'
+
+def install_system_gem(host, gem_name)
+  step "Install '#{gem_name}' gem"
+  # Ensure it's the system ruby
+  on(host, "/usr/bin/env gem install #{gem_name}")
```

Comment:
I *think* this is better than just calling `gem`, because the latter would depend on the PATH environment variable.  I don't *think* there will ever be a case where puppet would cause `/usr/bin/env` to return the puppet ruby, but I don't entirely know how `/usr/bin/env` is implemented.

I think what I would do is:
* Spin up a box.  make sure it has a system ruby and that that is the ruby you get via `/usr/bin/env`.
* Install puppet-agent on that box.  Log out and log back in and confirm that puppetlabs stuff has been added to PATH.
* Check `/usr/bin/env gem` again.  If that is still pointing to the system ruby then I think you're probably good with this approach.